src/main/java/com/based/basedsurvey/controller/WebQuestionController.java

```diff
@@ -16,7 +16,8 @@
 @Log
 @Controller
 public class WebQuestionController {
-    private final String MULTIPLE_CHOICE_SHORT = "MQC";
+    private final String MULTIPLE_CHOICE_SHORT = "MQC",
+    OPEN_ANSWER_QUESTION = "OAQ";
```

Comment:
it might make sense to use an enum here

Reply:
thymeleaf makes it really hard :(
i know https://www.baeldung.com/thymeleaf-enums exists, but it didn't work for me
I think it'd be more effort than its worth